client/clip_client/client.py

```diff
@@ -184,10 +181,15 @@ def _iter_doc(self, content) -> Generator['Document', None, None]:
                 )
 
     def _get_post_payload(self, content, kwargs):
+        parameters = {}
+        if 'batch_size' in kwargs:
+            parameters['minibatch_size'] = kwargs['batch_size']
```

Comment:
i disagree the exposing `minibatch_size` to public client. It can easily overload a CAS server. Imagine user now has the capability of controlling both `request_size` and `minibatch_size`, the user can easily occupy the full GPU usage on our Berlin GPU server. It can easily make our GPU OOM by setting large `request_size` and `minibatch_size`

Reply:
In C-S architecture, one should not aim to expose every server args to client, it is very risky.

Reply:
I see, that makes sense. Then we need to update the document about how to control batch size.